src/cb_admin/pages/AnonUser/CreateAnonUserForm.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+import { BirthYear } from '../../../shared/constants';
+import NavHeader from '../../../shared/components/NavHeader';
+import LabelledInput from '../../../shared/components/form/LabelledInput';
+import LabelledSelect from '../../../shared/components/form/LabelledSelect';
+import { FlexContainerCol } from '../../../shared/components/layout/base';
+import { Form, FormSection, PrimaryButton } from '../../../shared/components/form/base';
+import { Paragraph } from '../../../shared/components/text/base';
+
+const SubmitButton = styled(PrimaryButton)`
+  height: 4em;
+  width: 90%;
+`;
+
+const CenteredParagraph = styled(Paragraph)`
+  width: 90%;
+  font-size: 1.1em;
+  text-align: center;
+`;
+
+const CreateAnonUserForm = props => (
+  <FlexContainerCol>
```

Comment:
If possible I think we should try to move away from our home-rolled layout system towards using `react-flexbox-grid` (I've been trying to do this in my recent PRs). I remember you asked for a quick run through about what I was doing, still happy to do that.